src/ducks/Studio/defaults.js

```diff
@@ -17,33 +16,17 @@ export const DEFAULT_SETTINGS = {
   timeRange: DEFAULT_TIME_RANGE
 }
 
+export const COMMON_CHART_OPTIONS = {
+  isWatermarkLighter: getSavedToggle('isWatermarkLighter', false),
+  showWatermark: getSavedToggle('showWatermark', true),
```

Comment:
Variable name starts with the verb `show`, which implies that it's a function. Boolean variables should be named in the same style as the variables above and below (LOC 20, 22) - `isWatermarkVisible`